lasagne/utils.py

```diff
@@ -5,17 +5,40 @@
 
 
 def floatX(arr):
-    """
-    Shortcut to turn a numpy array into an array with the
-    correct dtype for Theano.
+    """Converts numpy array to one with the correct Theano data-type.
```

Comment:
You're right. It's explained in the "returns" segment what exactly this dtype is.